lab-matt/html/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html ng-app='cowsayApp'>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lab 21-22</title>
+    <link rel='stylesheet' type = 'text/css' href = 'bundle.css'/>
+    <script src='bundle.js'></script>
+  </head>
+  <body>
+    <main ng-controller = 'CowsayController'>
```

Comment:
This is sort of up to interpretation, but I think the idea of having a button with type="submit" is so that you can use the ng-submit directive. In the form tag, just add ng-submit="final()", then you can delete the ng-click on the submit button below. Doing this helps clarify what the form is doing.